### PR TITLE
feat: add ga and cookie banner

### LINKED ISSUE #12: Add google analytics tracking

### LINKED ISSUE #13: add cookie banner
@hagoptaminian, @hasanagh is going to implement this as he's doing it for the Graasp Desktop download page too. He will send you the PR for approval.

### PR DESCRIPTION
**Pull request by hasanagh**
adds ga and cookie banner

closes #13, closes #12

### CODE REVIEW

File: src/components/common/Layout/Layout.js [hunk 1]
```diff
@@ -17,6 +18,19 @@ const Layout = ({ children }) => (
...
         <GlobalStyles />
         {children}
+        <CookieConsent
+          location="bottom"
+          buttonText="Accept"
+          cookieName="gatsby-gdpr-google-analytics"
+          buttonStyle={{ background: '#fafafa', fontSize: '13px' }}
+          sameSite="lax"
+        >
+          We use cookies and other tracking technologies to improve your
```
Comment (juancarlosfarah) [inline src/components/common/Layout/Layout.js:28]: @hasanagh, I believe we already have EN and FR translation activated. Please check with @hagoptaminian to make this text available in both languages. Thanks!

File: gatsby-config.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 const path = require('path');
 
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
```
Comment (juancarlosfarah) [inline gatsby-config.js:4]: Just to confirm, we are using `.env.dev` and `.env.prod`, right?

File: package.json [hunk 3]
```diff
@@ -39,6 +42,7 @@
   "scripts": {
     "build": "gatsby build",
     "dev": "gatsby develop",
+    "serve": "gatsby serve",
```
Comment (juancarlosfarah) [inline package.json:45]: Why do we need this?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks Hassan! Just a minor comment and a couple questions.

File: src/components/common/Layout/Layout.js (hunk 1 above)
Comment (hagoptaminian) [reply src/components/common/Layout/Layout.js:28]: Correct - you can just `import { useTranslation }` from `react-i18next`, use `const { t } = useTranslation()` at the top of this component, and then wrap both `buttonText` and the cookie text with `{t('')}`. Make sure to update the translation pairs in `/src/config/i18n`.

**Comment by hagoptaminian** [comment]
@hasanagh this looks good to me. just needs the translation fix mentioned above

File: package.json (hunk 3 above)
Comment (hasanagh) [reply package.json:45]: Serve is mainly used when testing production builds, in my case I used it to test a build

File: gatsby-config.js (hunk 2 above)
Comment (hasanagh) [reply gatsby-config.js]: It wasn't, I just added a commit that does so.

**Review by hagoptaminian — APPROVED** [review]

**Pull request merged**